Bug Reporting/ARKEmailBugReporter.m

```diff
@@ -321,16 +247,139 @@ - (void)_stealFirstResponder;
 
 - (void)_showBugTitleCaptureAlert;
 {
-    UIAlertView *bugTitleCaptureAlert = [[UIAlertView alloc] initWithTitle:@"What Went Wrong?" message:@"Please briefly summarize the issue you just encountered. You’ll be asked for more details later." delegate:self cancelButtonTitle:@"Cancel" otherButtonTitles:@"Compose Report", nil];
-    bugTitleCaptureAlert.alertViewStyle = UIAlertViewStylePlainTextInput;
+    /*
+     iOS 8 often fails to transfer the keyboard from a focused text field to a UIAlertView's text field.
+     Transfer first responder to an invisble view when a debug screenshot is captured to make bug filing itself bug-free.
+     */
+    [self _stealFirstResponder];
+    
+    NSString * const title = NSLocalizedString(@"What Went Wrong?", nil);
+    NSString * const message = NSLocalizedString(@"Please briefly summarize the issue you just encountered. You’ll be asked for more details later.", nil);
+    NSString * const composeReportButtonTitle = NSLocalizedString(@"Compose Report", nil);
+    NSString * const cancelButtonTitle = NSLocalizedString(@"Cancel", nil);
     
-    UITextField *bugTitleTextField = [bugTitleCaptureAlert textFieldAtIndex:0];
-    bugTitleTextField.autocapitalizationType = UITextAutocapitalizationTypeSentences;
-    bugTitleTextField.autocorrectionType = UITextAutocorrectionTypeYes;
-    bugTitleTextField.spellCheckingType = UITextSpellCheckingTypeYes;
-    bugTitleTextField.returnKeyType = UIReturnKeyDone;
+#ifdef __IPHONE_8_0
```

Comment:
We actually still need the if ([UIAlertController class]) here. Otherwise this code will crash on iOS 8 when Aardvark is built from Xcode 7.

(also made the same comment on the commit itself, but figure I'd put it here to keep the conversation in the same thread)